sdk/agrifood/Azure.Verticals.AgriFood.Farming/src/Generated/InsightAttachments.cs

```diff
@@ -620,7 +622,6 @@ internal HttpMessage CreateCreateOrUpdateRequest(string partyId, string modelId,
             uri.AppendQuery("api-version", _apiVersion, true);
             request.Uri = uri;
             request.Headers.Add("Accept", "application/json");
-            request.Headers.Add("Content-Type", "multipart/form-data");
             request.Content = content;
```

Comment:
I'm curious, how we are going to set content-type then?

Reply:
We haven't decided yet, so I have disabled multipart support for DPG for now, cause we still have to decide about final implementation.
Feature will work only for protocol methods in HLC, this change will be reverted.